owncloudComLibrary/src/main/java/com/owncloud/android/lib/resources/status/RemoteCapability.kt

```diff
@@ -0,0 +1,105 @@
+/* ownCloud Android Library is available under MIT license
+ *   @author masensio
+ *   @author David González Verdugo
+ *   Copyright (C) 2019 ownCloud GmbH.
+ *
+ *   Permission is hereby granted, free of charge, to any person obtaining a copy
+ *   of this software and associated documentation files (the "Software"), to deal
+ *   in the Software without restriction, including without limitation the rights
+ *   to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ *   copies of the Software, and to permit persons to whom the Software is
+ *   furnished to do so, subject to the following conditions:
+ *
+ *   The above copyright notice and this permission notice shall be included in
+ *   all copies or substantial portions of the Software.
+ *
+ *   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ *   EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ *   MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ *   NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS
+ *   BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+ *   ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ *   CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ *   THE SOFTWARE.
+ *
+ */
+package com.owncloud.android.lib.resources.status
+
+/**
+ * Contains data of the Capabilities for an account, from the Capabilities API
+ */
+class RemoteCapability {
+    var accountName: String? = null
+
+    // Server version
+    var versionMayor: Int = 0
+    var versionMinor: Int = 0
+    var versionMicro: Int = 0
+    var versionString: String? = null
```

Comment:
All these nullable attributes are initialized later in the init block.

Why don't you remove the condition of nullable and convert them to not nullable?

Reply:
Done